src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1930,6 +1947,20 @@ private void VerifyChange(ReplicatorStatusChangedEventArgs change, int errorCode
                 _repl.AddDocumentReplicationListener(documentReplicated);
             }
 
+            if (isConflictResolvingFailed) {
+                _repl.AddDocumentReplicationListener((sender, args) =>
+                {
+                    if (!args.IsPush) {
+                        foreach(var d in args.Documents) {
+                            d.Error.Should().NotBeNull();
+                            var error = d.Error.As<CouchbaseException>();
+                            error.Error.Should().BeGreaterThan(0);
+                            error.Domain.Should().NotBeNull();
+                        }
+                    }
+                });
+            }
+
```

Comment:
This is not what I had in mind....once again this is throwing exceptions (via asserts) inside of a callback.  This passes now, but there is no way to detect if it fails....